nanopi/duo.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Richard Hull
+# See LICENSE.md for details.
+
+'''
+Created on Dec 31, 2017
```

Comment:
No need to include created date & author - this can be obtained from `git log`, but worthwhile setting docstring header to something like:

```
Alternative pin mappings for NanoPi DUO board (see http://...)
```